staging/src/k8s.io/kubectl/pkg/cmd/delete/delete_flags.go

```diff
@@ -73,9 +72,6 @@ func (f *DeleteFlags) ToOptions(dynamicClient dynamic.Interface, streams generic
 	if f.AllNamespaces != nil {
 		options.DeleteAllNamespaces = *f.AllNamespaces
 	}
-	if f.Cascade != nil {
-		options.Cascade = *f.Cascade
```

Comment:
Since other commands use DeleteFlags.ToOptions, is it possible to parse the propagation option here?